plugins/reporters/spdx/src/main/kotlin/Extensions.kt

```diff
@@ -288,11 +288,8 @@ internal fun OrtResult.getSpdxFiles(
             ),
             filename = fileFindings.path,
             licenseConcluded = SpdxConstants.NOASSERTION,
-            licenseInfoInFiles = fileFindings.licenses.takeIf { it.isNotEmpty() }?.map { it.toString() }
-                ?: listOf(SpdxConstants.NONE),
-            copyrightText = fileFindings.copyrights.sorted().joinToString("\n").takeUnless {
-                it.isBlank()
-            } ?: SpdxConstants.NONE
+            licenseInfoInFiles = fileFindings.licenses.ifEmpty { setOf(SpdxConstants.NONE) }.map { it.toString() },
```

Comment:
Maybe put `iEmpty` at the end to avoid the unnecessary `SpdxConstants.NONE.toString()` and to be more similar to the line below.